diagnostic.md

```diff
@@ -23,7 +38,7 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+$('.listenerItem').find('form')
```

Comment:
You would need to traverse up the DOM before finding the form because it is not a child of the `listerItem`.